code/modules/research/techweb/all_nodes.dm

```diff
@@ -1532,6 +1532,8 @@
 		"mod_mag_harness",
 		"mod_pathfinder",
 		"mod_holster",
+		"mod_criminal_capture",
```

Comment:
criminal capture cannot be printable while it gives you three free prisoner bags

Reply:
You want me to make a new child mod that doesn't give capture bags?

Reply:
Either that or limit the criminal capture in some way to require the modsuit to be nearby or something like that